src/scripts/main.js

```diff
@@ -1,3 +1,28 @@
 'use strict';
 
-// write code here
+const listEl = document.querySelector('ul');
+
+const convertToNumber = (str) => Number(str.replace('$', '').replace(',', ''));
+
+const sortList = (list, sortParam) =>
+  list.sort(
+    (a, b) =>
+      convertToNumber(b.dataset[sortParam]) - convertToNumber(a.dataset[sortParam]),
+  );
+
+const getEmployees = (sortedList, element) => (element.innerHTML = sortedList);
```

Comment:
The `getEmployees` function currently modifies the `innerHTML` of the provided element. This approach might lead to potential cross-site scripting (XSS) attacks. Consider creating and appending DOM elements instead of setting `innerHTML` directly.